code/game/objects/items/toys.dm

```diff
@@ -995,6 +995,47 @@
 	name = "tuxedo cat plushie"
 	icon_state = "tuxedocat"
 
+/obj/item/toy/plushie/greyplushie
+	name = "grey plushie"
+	desc = "A plushie of a grey wearing a sweatshirt. As a part of the 'The Alien' series, the doll features a sweater, an oversized head, and cartoonish eyes."
+	icon_state = "plushie_grey"
+	item_state = "plushie_grey"
+	var/hug_cooldown = FALSE //Defaults the plushie to being off coolodown. Sets the hug_cooldown var.
+	var/scream_cooldown = FALSE //Defaults the plushie to being off cooldown. Sets the scream_cooldown var.
+	var/singed = FALSE
+
+/obj/item/toy/plushie/greyplushie/water_act(volume, temperature, source, method = REAGENT_TOUCH) //If water touches the plushie the following code executes.
+	. = ..()
+	singed = TRUE
+	if(!singed)
+		return
+	icon_state = "grey_singed"
+	item_state = "grey_singed"//If the plushie gets wet the sprite changes to a singed version.
+	desc = "A ruined plushie of a grey. It looks like someone ran it under some water."
```

Comment:
Sorry, I should have been more clear here, this is what I meant:
```dm
if(!singed)
    singed = TRUE
    icon_state = "grey_singed"
	item_state = "grey_singed"//If the plushie gets wet the sprite changes to a singed version.
	desc = "A ruined plushie of a grey. It looks like someone ran it under some water."
```
This way, we only set the singed, icon and description vars when it's *not* already singed.

Reply:
Latest commit adds this but with an early return. It should ensure this isn't an issue.